_includes/templates/compatibility-page.md

```diff
@@ -1,3 +1,5 @@
+![{{tool.name|escape_once}}]({{tool.logo}}){:.third-party-logo}
```

Comment:
Do we definitely need the wallet name if we have the logo? For example, if I look at the Abra page, it looks a bit messy to have the name in the top left, then the logo (which is the name in stylized text) in the center, and then back to the left for the description.